wagtail_localize/test/models.py

```diff
@@ -236,7 +234,7 @@ def get_translatable_segments(self, value):
         return [StringSegmentValue("foo", "{} and some extra".format(value))]
 
 
-SF_KWARGS = {"use_json_field": True} if WAGTAIL_VERSION >= (4, 0) else {}
+SF_KWARGS = {"use_json_field": True}
```

Comment:
this should be removed and usages of `SF_KWARGS` should be replaced with `use_json_field=True`

Reply:
Removed SF_KWARGS, replaced with `use_json_field=True`